ScreenControl/Assets/ScreenControl/Core/Scripts/Interactions/InteractionModules/GrabInteraction.cs

```diff
@@ -216,7 +216,7 @@ private void OnDrawGizmos()
                 Gizmos.color = Color.red;
                 Gizmos.DrawWireSphere(-GlobalSettings.virtualScreen.PhysicalScreenPlane.normal * GlobalSettings.virtualScreen.PhysicalScreenPlane.distance, 0.01f);
                 Gizmos.color = Color.green;
-                Gizmos.DrawWireSphere(planeHit_debug + (-GlobalSettings.virtualScreen.VirtualScreenPlane.normal * GlobalSettings.virtualScreen.VirtualScreenPlane.distance), 0.01f);
+                //Gizmos.DrawWireSphere(planeHit_debug + (-GlobalSettings.virtualScreen.VirtualScreenPlane.normal * GlobalSettings.virtualScreen.VirtualScreenPlane.distance), 0.01f);
```

Comment:
Why comment this out instead of delete it?

Reply:
If we wanted to keep the gizmos in the examples in future, we may want a reference for the gizmo that I have broken by deleting 'GlobalSettings.virtualScreen.VirtualScreenPlane'